.github/workflows/black.yml

```diff
@@ -8,4 +9,4 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
-      - uses: psf/black@stable
+      - uses: psf/black@master
```

Comment:
Why is this changed?

Reply:
Oh because of "Error: Unable to resolve action `psf/black@stable`, unable to find version `stable`". I think it would be better to pin a version instead of master? Not sure.

Reply:
Because black changed their main branch and broke all the CIs of the world.

Check the output of the job of commit a5931d7  -> https://github.com/kayak/pypika/runs/2277687520

Reply:
Yeah I noticed. If using master is the recommended approach then I'm fine with it.